core/src/main/java/org/apache/carbondata/core/metadata/PartitionMapFileStore.java

```diff
@@ -279,18 +279,20 @@ public boolean isPartionedSegment() {
    *                      dropped
    * @throws IOException
    */
-  public void dropPartitions(String segmentPath, List<String> partitionsToDrop, String uniqueId,
-      boolean partialMatch) throws IOException {
+  public void dropPartitions(String segmentPath, List<List<String>> partitionsToDrop,
```

Comment:
All the callers right now has passed only single list for "partitionsToDrop". So I think we can make single list. Is there a special purpose?